source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRColumnRenderer.mm

```diff
@@ -99,6 +99,7 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
                                                                   recipientView:column
                                                                   actionElement:acoSelectAction
                                                                      hostConfig:acoConfig];
+    column.shouldGroupAccessibilityChildren = YES;
```

Comment:
Are we also doing the same thing for selectActions on ColumnSet and Container?

Reply:
Yes, they will be addressed.